patches/0002-Add-crypto-backend-foundation.patch

```diff
@@ -396,10 +631,10 @@ index 00000000000000..e5d7570d6d4363
 +const isRequireFIPS = true
 diff --git a/src/crypto/internal/backend/nobackend.go b/src/crypto/internal/backend/nobackend.go
 new file mode 100644
-index 00000000000000..f45f2fe76d54f1
+index 00000000000000..db9ec456db705e
 --- /dev/null
 +++ b/src/crypto/internal/backend/nobackend.go
-@@ -0,0 +1,154 @@
+@@ -0,0 +1,192 @@
 +// Copyright 2017 The Go Authors. All rights reserved.
```

Comment:
Is it normal to add this copyright header to our patches? As opposed to a Microsoft version?

Reply:
Our philosophy is to try to upstream everything we add in the Go source tree. Using the Go authors copyright simplifies the upstreaming process.

We need to document our copyright guidelines...